### DISCUSSION TITLE
[운영체제 2일차] 프로세스

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#85] [오늘의 주제]
프로세스 질문 2인조 1개, 3인조 2개

**Comment by zini9188** [comment]
프로세스간 통신이란 무엇이고 이를 위한 주요 방법이 무엇이 있는지 설명해주세요

**Comment by zini9188** [comment]
컨텍스트 스위칭에 대해 설명하고, 프로세스 간에는 어떻게 진행되는지 설명해주세요.

**Comment by watty-exclamationMark** [comment]
프로세스의 구조를 말해주세요.
그리고 프로세스와 프로그램의 차이점을 말해주세요.

**Reply by watty-exclamationMark** [reply]
+)
프로세스는 PCB를 가지고 있는 객체입니다.
PCB(Process contropl block)은 무엇이고 PCB가 하는 일은 무엇일까요?

**Comment by kjc6735** [comment]
프로세스 종류는 무엇이 있는지 설명해주세요

**Comment by seongeun42** [comment]
프로세스의 생성 과정에 대해 설명해주세요.
